Meta/remove_unused_includes.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
```

Comment:
Nope, I eventually gave up because the effect on compilation time was too little.

Reply:
Also, yes, in fact: Please use "env" to figure out which python to run, as a hardcoded path will break for every user using virtualenv/pypoetry/silimar. Also, I'm sure that there's at least one OS which installs python under /bin, but not /usr/bin.

Reply:
Regarding speed: Once upon a time I removed a significant amount of unused headers (and could prove it!) in https://github.com/SerenityOS/serenity/pull/16829 . In the section `How fast?` , drop-down "Measurements", you can see that the compilation time drops from 3016 to 2996. That's a measly 0.66% speed-up. My best guess is that with this method, the total speedup can be around 2%, at most.

I'm sure that other methods of speeding things up are more effective, have less effort, and last longer. For example splitting up AK/Stream.h looks promising to me, because it is included by *everything, everywhere*, and most places use barely any of the definitions.